app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -63,6 +63,11 @@ def create
       end
     end
 
+    def destroy
+      @admin_set.destroy
+      redirect_to sufia.admin_admin_sets_path, notice: 'Administrative set successfully deleted'
```

Comment:
Can you:

- [ ] i18nize this string for English? 
- [ ] create a new Hyrax ticket for making sure to add a Spanish ticket once this work is merged over in Hyrax?